test/AllInOne.Formatted.cs

```diff
@@ -749,6 +749,40 @@ public void ConstructedType()
             List<int> i = null;
             int c = i.Count;
         }
+
+        class Foo : IDisposable
+        {
+            public int N;
+
+            public void Dispose()
+            {
+            }
+        }
+
+        unsafe void EmptyEmbeddedStatment()
+        {
+            // useless empty statements get deleted
+
+
+            // but nontrivial ones remain
+            if (true)
+                ;
+            else if (true)
+                ;
+            else
+                ;
+            while (false) ;
```

Comment:
Spaces and indents look inconsistent. It's unrelated to this review so I'll look at it after merging it.